inbm/dispatcher-agent/dispatcher/source/source_command.py

```diff
@@ -113,16 +112,29 @@ def _handle_app_source_command(
         return Result(status=200, message=serialized_list)
 
     if "remove" in app_action:
-        keyname = parsed_head.get_children("applicationSource/remove/gpg")["keyname"]
+        keyname = None
+        try:
+            keyname = parsed_head.get_children("applicationSource/remove/gpg")["keyname"]
+        except XmlException:
```

Comment:
Add error handling for the case where the manifest does not contain the expected GPG key elements, to provide a clear error message to the user. [important]

Reply:
nope.  We want to let this error be ignored.